src/backend/joanie/tests/core/test_api_contract.py

```diff
@@ -818,6 +854,9 @@ def test_api_contract_generate_zip_archive_authenticated_post_with_no_signed_con
 
         self.assertEqual(response.json(), ["No zip to generate"])
 
+    @override_settings(
+        JOANIE_SIGNATURE_BACKEND="joanie.signature.backends.dummy.DummySignatureBackend"
+    )
```

Comment:
What do you think to set this value on `Test` configuration ?

Reply:
Yes, it would simplify the tests.
But I suggest to do it in another PR, as this one is already pretty big, WDYT ?

Reply:
I did it here https://github.com/openfun/joanie/pull/489/commits/204f6d5c98e953a2dd3811a85963a5005a068ba6 as it was small.